TinyDPS.lua

```diff
@@ -2379,7 +2379,7 @@ local function newBar(g)
   dummybar:SetWidth(tdpsFrame:GetWidth() - 4)
   dummybar:SetHeight(tdps.barHeight)
   dummybar:Hide()
-  --dummybar:SetPoint("RIGHT", tdpsFrame, "RIGHT", -2, 0)
```

Comment:
I'm not sure why this was commented-out.  TidyBar seems to work with it uncommented.